### DISCUSSION TITLE
Metrics for templates [weight]

### DISCUSSION THREAD

**Discussion by pdelteil** [projectdiscovery/nuclei#1475] [Ideas]
**Is your feature request related to a problem? Please describe.**
I've notice that templates have different 'weight' depending on how many requests they perform. This might be impactful for an automated process and probably some people would like to exclude 'heavy' templates that might provide low impact or low rewards. 

**Describe the solution you'd like**
It would be good to have a way of sorting the templates according to the amount of requests they perform, this could be a first measure of how heavy the template is.

**Comment by TerminalFi** [comment]
Sounds like you would like a flag / option for `ClusterMaxRequests` in which you can filter out templates/workflows that have X amount of requests or more.

Correct?

**Comment by forgedhallpass** [comment]
Usually fuzzer/dos templates make multiple requests, but they are not enabled by default. If you still want to run them, but exclude info/low severity issues just run a separate command with the fuzz and dos tags included and set the severity to medium,high,critical.
